src/v/security/audit/audit_log_manager.cc

```diff
@@ -18,7 +18,7 @@
 #include "kafka/client/config_utils.h"
 #include "kafka/protocol/produce.h"
 #include "kafka/protocol/schemata/produce_response.h"
-#include "kafka/server/handlers/topics/types.h"
+#include "kafka/protocol/topic_properties.h"
```

Comment:
these names are conceptually part of the protocol. this change removes the dependency on kafka server which is problematic since kafka server depends on audit.